components/tekton/tasks/buildah.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/categories: Image Build
+    tekton.dev/pipelines.minVersion: 0.17.0
+    tekton.dev/platforms: linux/amd64,linux/s390x,linux/ppc64le,linux/arm64
+    tekton.dev/tags: image-build
+  labels:
+    app.kubernetes.io/version: "0.5"
+  name: buildah
```

Comment:
Any reason for not using the buildah `ClusterTask`?

Reply:
After a chat about this with Jose:
```
If you change version the buildah version may change and potentially break everything
```

So we decided to maintain our own version of buildah task

Reply:
Was this reported as a BZ or Jira to fix what is wrong?

Reply:
@jnunyez

Reply:
@novacain1 It is not that the the ClusterTask is now **wrong** but didnt totally fit with our needs. First, the ClusterTask was on the one hand coupled to docker. Maybe we can PR to tekton and propose some buildah variant.  
Also as you mentioned it uses the non supported image and changed that.

Reply:
sounds good.